test/development/app-dir/ssr-only-error/ssr-only-error.test.ts

```diff
@@ -40,4 +41,22 @@ describe('ssr-only-error', () => {
      }
     `)
   })
+
+  it('should not handle internal nextjs errors that will be handled by error boundaries', async () => {
+    const browser = await next.browser('/notfound', {
+      pushErrorAsConsoleLog: true,
+    })
+
+    await assertNoRedbox(browser)
+    expect(await hasErrorToast(browser)).toBe(false)
+
+    const text = await browser.elementByCss('body').text()
+    expect(text).toBe('404\nThis page could not be found.')
+
+    expect(await browser.log()).not.toContainEqual(
+      expect.objectContaining({
+        source: 'error',
```

Comment:
Is this correct? Shouldn't there be at least one error due to 404?

Reply:
This message was brought to you by your personalized test runner: https://github.com/vercel/next.js/actions/runs/13376531235/job/37356943350?pr=76124#step:33:262

Let's just filter for `source: 'error'` and no 404 and then assert that this obj is empty. Makes it more obvious.